manifests/script-lab-edge.olk.xml

```diff
@@ -0,0 +1,330 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You no longer need "edge" and "insider" manifests, you can delete them and only leave the prod + localhost.
We support in-app environment switching now, so the single prod entry is enough for any of the non-dev environments.